### ISSUE TITLE
Exceptions per Site/(Sub-)Domain

### ISSUE THREAD

**Issue by chainria** [nextcloud/passman#618]
## Feature request

**User type**: Logged-in user

**User level**: intermediate or advanced

### Description

When there are a lot of passwords, the "ignore subdomains" etc. is not sufficient anymore. For instance, I have a mixed bag of subdomains which DO share passwords and then there are plenty which do NOT do this.

### Benefit / value

Right now Passman is hard to use on domains which have multiple subdomains which each need their credentials. In Lastpass there is the option of Exceptions per Site. This would be a very great feature for PassMan as well.

### Risk / caveats

Other that there may be some more complex handling, I cannot think of any. It's a feature which can be used, but most users won't even need it I guess.

### Sponsorship

 
**Are you a developer willing to implement this feature?**: no

I can assist with testing or write some patches eg, but I cannot develop this feature myself.

**Can you sponsor the development of this feature or do you know someone who can?**: no

**Comment by newhinton** [comment]
@chainria Isn't that a feature for the webextension? Please recreate this issue over at https://github.com/nextcloud/passman-webextension/issues

Thank you!

**Comment by chainria** [comment]
Oh yes, thank you! Wasn't aware of that. But Isn't it something that needs to be changed in the core as well? Whatever the case will be, I'll repost the issue over there.

**Comment by Oliv4945** [comment]
@newhinton : yes and no. Both are involved as the webextension can only apply a rule stored in database ;)

**Comment by newhinton** [comment]
Okay, now i got your problem! Yes, i see where that can be useful, but i have to think whether that belongs to the core or only the webextension. So for now we keep both open

**Comment by Oliv4945** [comment]
It can be in extension settings only but would be per computer which is I prefer to avoid.
If this kind of feature is implemented it would be great to filter also ports 😃

**Comment by newhinton** [comment]
I see, maybe we can reuse the custom fields and introduce an url type which can be used by the extension. @Oliv4945 would you be up to help out with the extension?
